readme/LinuxBuild.md

```diff
@@ -1,108 +1,94 @@
-```bash
+```sh
 sudo dnf install gcc-c++ cmake gtk3-devel libxml2-devel cppunit-devel portaudio-devel libsndfile-devel \
-poppler-glib-devel texlive-scheme-basic texlive-dvipng 'tex(standalone.cls)' gettext libzip-devel \
-librsvg2-devel gtksourceview4-devel
+  poppler-glib-devel texlive-scheme-basic texlive-dvipng 'tex(standalone.cls)' gettext libzip-devel \
+  librsvg2-devel gtksourceview4-devel
 ```
 
-
 #### For Ubuntu/Debian and Raspberry Pi OS:
-````bash
+```sh
 sudo apt-get install cmake libgtk-3-dev libpoppler-glib-dev portaudio19-dev libsndfile-dev \
-dvipng texlive libxml2-dev liblua5.3-dev libzip-dev librsvg2-dev gettext lua-lgi \
-libgtksourceview-4-dev
-````
+  dvipng texlive libxml2-dev liblua5.3-dev libzip-dev librsvg2-dev gettext lua-lgi \
+  libgtksourceview-4-dev
+```
 
 #### For openSUSE:
-```bash
+```sh
 sudo zypper install cmake gtk3-devel portaudio-devel libsndfile-devel \
-texlive-dvipng texlive libxml2-devel libpoppler-glib-devel libzip-devel librsvg-devel lua-devel lua-lgi \
-gtksourceview4-devel
+  texlive-dvipng texlive libxml2-devel libpoppler-glib-devel libzip-devel librsvg-devel lua-devel lua-lgi \
+  gtksourceview4-devel
 ```
 
 #### For Solus:
-```bash
+```sh
 sudo eopkg it -c system.devel
 sudo eopkg it cmake libgtk-3-devel libxml2-devel poppler-devel libzip-devel \
-portaudio-devel libsndfile-devel alsa-lib-devel lua-devel \
-librsvg-devel gettext libgtksourceview-devel
+  portaudio-devel libsndfile-devel alsa-lib-devel lua-devel \
+  librsvg-devel gettext libgtksourceview-devel
 ```
 
-## Compiling
+## Building and Testing
 
-The basic steps to compile Xournal++ are:
+See [Compile.md](./Compile.md)
 
-```bash
-git clone http://github.com/xournalpp/xournalpp.git
-cd xournalpp
-mkdir build
-cd build
-cmake ..
-cmake --build .
-# For a faster build, set the flag -DCMAKE_BUILD_TYPE=RelWithDebInfo
-```
+## Building Documentation
 
-Use `cmake-gui ..` to graphically configure compilation.
+The code documentation is generated using Doxygen. Both Doxygen and graphviz must be
+installed. For example, with apt:
 
-With Cairo 1.16 PDF Bookmarks will be possible, but this Version is not yet
-common available, therefore the Cairo PDF Export is without PDF Bookmarks.
+```sh
+sudo apt install doxygen graphviz
+```
 
-The binary executable will be placed directly into the `build/` directory. 
-You can run it by executing `./xournalpp` from that directory. For testing
-purposes packaging and installation on the system are not required.
+Then, execute `doxygen` in the root directory of the repository. The documentation
+can be found in `doc/html` and `doc/latex`. Conveniently view it in a browser with
+`python3 -m http.server 8000` and visit the URL it shows.
 
 ## Packaging and Installation
 
-### Creating Packages for Package Managers
+### Packaging
 
 Please ensure that the `translations` target has been built before
 attempting to generate any package.
-```bash
+```sh
 cmake --build . --target translations
 ```
```

Comment:
This is outdated and can be removed; the translations target will always run for the build target.

Reply:
Ah, I was wondering about that - I'll make those changes :+1: